src/rpc/types.rs

```diff
@@ -23,9 +23,34 @@ pub struct Status {
     // pub throughput: f64,
 }
 
+#[derive(Debug, Clone, Default, PartialEq)]
+pub enum RouteGroup {
+    #[default]
+    Ethereum,
+    StrataCL,
+}
+
+impl RouteGroup {
+    pub fn from_method_name(method_name: &str) -> Self {
+        if method_name.starts_with("alp_") {
+            Self::StrataCL
+        } else {
+            Self::Ethereum
+        }
+    }
```

Comment:
I'd probably break this down and make it return an error if it fails to parse, so we can handle the fallback higher up the chain.  Ethereum RPC names are [pretty well-defined](https://ethereum.org/en/developers/docs/apis/json-rpc/).

Reply:
I am thinking of ethereum rpcs as the 'default' and the others as special overrides. I'd like to keep this simple without explicitly listing all ethereum rpcs here.